frontend/routes/viewerGui/components/reportedItems/reportedItems.component.tsx

```diff
@@ -200,6 +202,7 @@ export class ReportedItems extends React.PureComponent<IProps, IState> {
 		const showDefaultHiddenItemsChanged = prevProps.showDefaultHiddenItems !== showDefaultHiddenItems;
 		const searchEnabledChange = prevProps.searchEnabled !== searchEnabled;
 		const detailsWasClosed = prevProps.showDetails !== showDetails && !showDetails;
+		const detailsShowing =  prevProps.showDetails !== showDetails && showDetails;
```

Comment:
Doesn't look like this is used anywhere? @sanmont3drepo

Reply:
Yes, sorry, was a previous attempt.